ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/Util/Serialisation/Golden.hs

```diff
@@ -435,12 +434,8 @@ goldenTest_SerialiseNodeToClient codecConfig goldenDir Examples {..} =
 deriving instance Generic TermToken
 deriving instance ToExpr  TermToken
 
-instance Condense FlatTerm where
-  condense [] = "[]"
-  condense (x:ys) = intercalate "\n" $
-      [ "[ " <> show x ] <>
-      [ ", " <> show y | y <- ys ] <>
-      [ "]" ]
+instance Condense TermToken where
+  condense = show
```

Comment:
Why this change? The previous instance printed each term on a separate line.

Update: I see you changed some things related to the `Condense` instance for lists (for condensing a list of events).

Reply:
Yeah, avoided overlapping instances for `Condense`. We needed to `condense` a `[LedgerUpdate blk]`, which is a type family, and hence could not resolve the overlapping instances.